### PR TITLE
Skylar pm2 configuration

### PR DESCRIPTION
**Pull request by swordfish444**
@shyam-habarakada Added tests for api.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +28,7 @@
     "cors": "^2.0.0",
     "debug": "^2.0.0",
     "express": "^4.0.0",
+    "js-yaml": "^3.6.1",
```
Comment (shyam-habarakada) [inline package.json:30]: Dupe. See lines above

Comment (swordfish444) [inline package.json:30]: Thanks. Yeah we both implemented the config to yaml. We agreed I would do this.

**Comment by shyam-habarakada** [comment]
Is there something like FactoryGirl, for nose?

**Comment by shyam-habarakada** [comment]
node, not nose ;-)

**Pull request merged**